ajax/libs/ekko-lightbox/package.json

```diff
@@ -15,7 +15,7 @@
     "type": "git",
     "url": "https://github.com/ashleydw/lightbox.git"
   },
-  "license": {
+  "license": "MIT",
```

Comment:
Please modify `license` field to
```
  "license": "MIT"
```

Thanks!

Reply:
Please modify the commit message to be clearer. Like the current title.
And please mention reference and issue number in the body of the commit message. Like
```
Ref: https://github.com/ashleydw/lightbox/blob/master/LICENSE
cc #11931
```